### PR TITLE
[12.0][MIG] server environment from 11.0

### PR DESCRIPTION
**Pull request by sbidoul**
A trivial migration.

Includes https://github.com/OCA/server-tools/pull/1383

### CODE REVIEW

File: server_environment/README.rst [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        def _server_env_fields(self):
+            return {"directory_path": {}}
+
+Read the documentation of the class and methods in `models/server_env_mixin.py
+<models/server_env_mixin.py>`__.
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/149/10.0
```
Comment (simahawk) [inline server_environment/README.rst:154]: :fire: can you move README to new structure? :wink:

File: server_environment/__init__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+##############################################################################
```
Comment (simahawk) [inline server_environment/__init__.py:1]: I think we can finally cleanup this headers too...

File: server_environment_files_sample/README.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-GPL--3-blue.svg
```
Comment (simahawk) [inline server_environment_files_sample/README.rst:1]: same :)

**Review by simahawk — CHANGES_REQUESTED** [review]
can you include [12.0] in your commit? tnx!

File: server_environment/README.rst (hunk 1 above)
Comment (sbidoul) [reply server_environment/README.rst:154]: Yes of course. Done.

File: server_environment/__init__.py (hunk 2 above)
Comment (sbidoul) [reply server_environment/__init__.py:1]: @simahawk Okay, I can do that too. What license is it btw? The main module and manifest say GPL3+, but other headers here and there say AGPL.

Comment (simahawk) [reply server_environment/__init__.py:1]: interesting.... @guewen do you have a more clear idea?

Comment (guewen) [reply server_environment/__init__.py:1]: It seems the old files and the manifest are all stating "GPL-3 or any later version" and only my new mixin (and their tests) is set as AGPL-3, the inconsistency is my fault then. We can change the headers with AGPL3 to match the rest if this is the easy way.

File: server_environment/readme/CONTRIBUTORS.rst [hunk 4]
```diff
@@ -0,0 +1,10 @@
+* Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+* Daniel Reis <dgreis@sapo.pt>
+* Florent Xicluna <florent.xicluna@gmail.com>
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
+* Holger Brunn <hbrunn@therp.nl>
+* Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com>
+* Nicolas Bessi <nicolas.bessi@camptocamp.com>
+* Wingo
+* Yannick Vaucher <yannick.vaucher@camptocamp.com>
+* Stéphane Bidoul <stephane.bidoul@acsone.eu>
```
Comment (yvaucher) [inline server_environment/readme/CONTRIBUTORS.rst]: I don't like alphabetically sorted contributor list, even more when it's not sorted properly :grin: 

```
* Florent Xicluna (Wingo) <florent.xicluna@gmail.com>
* Nicolas Bessi <nicolas.bessi@camptocamp.com>
* Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
* Daniel Reis <dgreis@sapo.pt>
* Holger Brunn <hbrunn@therp.nl>
* Leonardo Pistone <leonardo.pistone@camptocamp.com>
* Adrien Peiffer <adrien.peiffer@acsone.com>
* Thierry Ducrest <thierry.ducrest@camptocamp.com>
* Guewen Baconnier <guewen.baconnier@camptocamp.com>
* Thomas Binfeld
* Stéphane Bidoule <stefan.bidoule@acsone.com>
```

IMO can be omited:

* Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com> (initial commit moving addon from lp:c2c-addons
* Sandy Carter: generated pot files on multiple addons?
* Pedro Baeza, moving files in `__unported__`
* Yannick Vaucher <yannick.vaucher@camptocamp.com> added icon, fixed some PEP8 and answered to one question at Odoo XP :wink: 


The contributor list can be smaller in the sample module.

**Review by yvaucher — APPROVED** [review]
Thanks !

The contributor list could be improved, but I'm not blocking for this.

**Comment by sbidoul** [comment]
@guewen @yvaucher copyright headers adapted (all GPL3) as well as contributors list..

**Comment by sbidoul** [comment]
@simahawk too :)

**Review by simahawk — APPROVED** [review]
LG

**Comment by sbidoul** [comment]
Set status "ready to merge" manually. For some reason the bot, which searches on status:success does not pick this one up, as GitHub thinks it is still pending.

**Pull request merged**